### PR TITLE
Enhancement/descriptive manifest validation

### PR DESCRIPTION
**Pull request by Anonycoders**

### CODE REVIEW

File: fandogh_cli/fandogh_client/__init__.py [hunk 1]
```diff
@@ -308,6 +311,16 @@ def deploy_manifest(manifest):
                              headers={'Authorization': 'JWT ' + token}
                              )
     if response.status_code != 200:
+        if response.status_code == 400:
+            keys = str(list(response.json().keys())[0]).split('.')
+            temp = x
```
Comment (MahdiZareie) [inline fandogh_cli/fandogh_client/__init__.py]: what is x?
please Improve variable namings

**Review by MahdiZareie — APPROVED** [review]

**Review by MahdiZareie — APPROVED** [review]

File: fandogh_cli/fandogh_client/__init__.py (hunk 1 above)
Comment (s-soroosh) [reply fandogh_cli/fandogh_client/__init__.py]: +1

File: fandogh_cli/fandogh_client/__init__.py [hunk 2]
```diff
@@ -308,6 +311,16 @@ def deploy_manifest(manifest):
...
+            for key in keys:
+                temp = temp.get(key, None)
+
+            if temp is not None:
```
Comment (s-soroosh) [inline fandogh_cli/fandogh_client/__init__.py:320]: what if temp is none?

Comment (Anonycoders) [reply fandogh_cli/fandogh_client/__init__.py:320]: If none -> normal error that has came from the server will be shown, like the old days :D

**Pull request closed without merging**